app/datatables/repository_datatable.rb

```diff
@@ -335,10 +323,10 @@ def sort_assigned_records(records, direction)
       ids = unassigned + assigned
     end
 
-    order_by_index = ActiveRecord::Base.send(
+    order_by_index = ActiveRecord::Base.__send__(
```

Comment:
I don't think that ActiveRecord::Base overrides Object.send method. Do you have a good reason to use this?

Reply:
Just linter was complaining about it

Reply:
fuck the linter this is not python :smile: